python/src/trezorlib/btc.py

```diff
@@ -223,6 +224,7 @@ def sign_tx(
             coin_name=coin_name,
             inputs_count=len(inputs),
             outputs_count=len(outputs),
+            decred_staking_ticket=decred_staking_ticket,
```

Comment:
this is useless, the field is covered by `kwargs` handling

Reply:
https://github.com/trezor/trezor-firmware/pull/1249/commits/20aa862004cd9731524b43026be377223078237c